pkg/connpool/dialer.go

```diff
@@ -161,7 +169,7 @@ func (h *dialer) sendTCD(ctx context.Context, code ControlCode) {
 func (h *dialer) readLoop(ctx context.Context) {
 	defer func() {
 		// allow write to drain and perform the close of the connection
-		dtime.SleepWithContext(ctx, 200*time.Millisecond)
+		dtime.SleepWithContext(ctx, 100*time.Millisecond)
```

Comment:
walking this back to its previews value as i think the new change is actually the right way to ensure the write drains